rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,193 @@
+---
+- name: pe
+  type: group
+  fields:
+    - name: main_icon
+      level: extended
+      type: object
+      description: >
+        Hashes of embedded program icon.
+      fields:
+        - name: dhash
+          level: extended
+          type: keyword
+          description: >
+            Difference Hash (dhash) for a given PE file.
+          example: b806e17c8e330d82
+
+        - name: md5
```

Comment:
Would it make sense to have the `icon` info contain a wider variety of subfields such as things like file type or size in bytes? Mainly just wondering, because if so, I feel like we should follow the form of other ECS fields here and move this to something like `pe.icon.hash.md5`

Reply:
yeah, `pe.icon` is already a group object, so adding `hash.` for different algos (to include `dhash`) would work.